components/controls/selectors/DateRangeSelector.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div>
+    <div>
+      <div class="row">
+        <div class="col-6">
+          <div style="position: absolute;right: -6px;top: 5px;">
+            to
+          </div>
+          <input
+            id="DateRange_StartDate"
+            v-model="selectedStartDate"
+            type="text"
+            class="form-control"
+            placeholder="Select start date"
+            :disabled="disabled"
```

Comment:
max should be set to endDate when enddate is selected:

```suggestion
            :disabled="disabled"
            :max="selectedEndDate"
```

Reply:
Done!